hugo/layouts/partials/head.html

```diff
@@ -22,7 +22,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
     <meta name="viewport" content="width=device-width, user-scalable=yes">
 
-    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700|Roboto:300,400,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700" rel="stylesheet">
```

Comment:
Maybe in just one line:

`https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700|Roboto:400,700`